packages/lib-utils/src/k8s/hooks/useDeepCompareMemoize.ts

```diff
@@ -0,0 +1,16 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+export const useDeepCompareMemoize = <T = unknown>(value: T, stringify?: boolean): T => {
```

Comment:
This was co-located with the hooks in the OpenShift Console SDK -- this can be more generically used. Not sure we want to export it though. But maybe as a utility method? Up to discussion.

Reply:
Hm... Did we take https://www.npmjs.com/package/use-deep-compare and modify it to play nice with typescript?

Reply:
Chatted with Andrew and agreed to create a follow-up investigation item: https://issues.redhat.com/browse/HAC-795

Reply:
@florkbr I think @rawagner wrote the initial impl. for this hook so he might know more about the intention behind it.